apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -184,6 +205,19 @@ type ApicastProductionSpec struct {
 	// +optional
 	// +kubebuilder:validation:Enum=debug;info;notice;warn;error;crit;alert;emerg
 	LogLevel *string `json:"logLevel,omitempty"` // APICAST_LOG_LEVEL
+	// CustomPolicies specifies an array of defined custome policies to be loaded
+	// +optional
+	CustomPolicies []CustomPolicySpec `json:"customPolicies,omitempty"`
+	// HttpsPort controls on which port APIcast should start listening for HTTPS connections. If this clashes with HTTP port it will be used only for HTTPS.
+	// +optional
+	HTTPSPort *int32 `json:"httpsPort,omitempty"` // APICAST_HTTPS_PORT
```

Comment:
Which one of this parameters is the one that controls whether TLS is enabled? `httpsPort`? It would be good to explain that behavior documented somewhere, or reorganize a little bit the structure in the CR to make it clear that "you are enabling tls"

Reply:
It is in the doc: https://github.com/3scale/3scale-operator/pull/632/files#diff-72dab7f3126dc7dbd85ab6398f36d48315960176cddc220c7fd9bdb36da97fb6R5

`Enable TLS at APIcast pod level setting either `httpsPort` or `httpsCertificateSecretRef` fields or both.`

Reply:
Maybe add it in the apimanager_type too then? :thinking:

Reply:
I will add it to the apimanager_type

Reply:
added